### DISCUSSION TITLE
Upper/Lower case character error in file-restart

### DISCUSSION THREAD

**Discussion by tonyzahtila** [Nek5000/nekRS#275] [General]
Hi all,

I thought I would offer this feedback. 

If I choose a restart file in turbPipe.par such as 'turbPipe0.f00027' (note the camel case), then I get an error that nek could not read 'turbpipe0.f00027', i.e, the upper case character has been modified to a lower case character.

Of course the simple fix I have used is just to copy my turbPipe0.f00027 to turbpipe0.f00027.

I just thought I would let you know.

Tony

**Comment by yslan** [comment]
Hi Tony,

And the double quotation mark can keep the case sensitive. 
For example, `startFrom = "turbPipe0.f00027"`

Yu-Hsiang
